app/app.iml

```diff
@@ -26,7 +27,7 @@
+          <option name="classpath" value="$USER_HOME$/.gradle/caches/transforms-1/files-1.1/appcompat-v7-25.4.0.aar/23891ea0530a3c707da5701faf1df265/jars/classes.jar:/Users/amandasjostrom/.gradle/caches/modules-2/files-2.1/org.apache.commons/commons-lang3/3.7/557edd918fd41f9260963583ebf5a61a43a6b423/commons-lang3-3.7.jar:/Users/amandasjostrom/.gradle/caches/modules-2/files-2.1/joda-time/joda-time/2.10/f66c8125d1057ffce6c4e29e624cac863e110e2b/joda-time-2.10.jar:/Users/amandasjostrom/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-jdk7/1.3.21/d207ce2c9bcf17dc8e51bab4dbfdac4d013e7138/kotlin-stdlib-jdk7-1.3.21.jar:/Users/amandasjostrom/.gradle/caches/transforms-1/files-1.1/animated-vector-drawable-25.4.0.aar/5113f7f99770c4940eda7d8e59945a49/jars/classes.jar:/Users/amandasjostrom/.gradle/caches/transforms-1/files-1.1/support-vector-drawable-25.4.0.aar/703652521370794e978f3b9fd6aeb98d/jars/classes.jar:/Users/amandasjostrom/.gradle/caches/transforms-1/files-1.1/support-v4-25.4.0.aar/106d004a3f636564f72cd8529e23f12d/jars/classes.jar:/Users/amandasjostrom/.gradle/caches/transforms-1/files-1.1/support-fragment-25.4.0.aar/5490e7854df3635f7e9f3bddbb3a096a/jars/classes.jar:/Users/amandasjostrom/.gradle/caches/transforms-1/files-1.1/support-media-compat-25.4.0.aar/d51e54c3e2b58f06d5a8af5fe0cc5762/jars/classes.jar:/Users/amandasjostrom/.gradle/caches/transforms-1/files-1.1/support-core-utils-25.4.0.aar/20362a41d57e33bb5d17d16f789a986b/jars/classes.jar:/Users/amandasjostrom/.gradle/caches/transforms-1/files-1.1/support-core-ui-25.4.0.aar/163f04a8dcfa8f2df80b3850b19f3139/jars/classes.jar:/Users/amandasjostrom/.gradle/caches/transforms-1/files-1.1/support-compat-25.4.0.aar/0dd2485c9317647631e1198ca0547290/jars/classes.jar:/Users/amandasjostrom/.gradle/caches/modules-2/files-2.1/com.android.support/support-annotations/25.4.0/f6a2fc748ae3769633dea050563e1613e93c135e/support-annotations-25.4.0.jar:/Users/amandasjostrom/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib/1.3.21/4bcc2012b84840e19e1e28074284cac908be0295/kotlin-stdlib-1.3.21.jar:/Users/amandasjostrom/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-common/1.3.21/f30e4a9897913e53d778f564110bafa1fef46643/kotlin-stdlib-common-1.3.21.jar:/Users/amandasjostrom/.gradle/caches/modules-2/files-2.1/org.jetbrains/annotations/13.0/919f0dfe192fb4e063e7dacadee7f8bb9a2672a9/annotations-13.0.jar:/Users/amandasjostrom/Library/Android/sdk/platforms/android-25/android.jar" />
```

Comment:
Hmm, this file should probably be removed from version control, since it contains personal file paths.